ansible/roles/ssc/tasks/main.yml

```diff
@@ -31,3 +31,33 @@
   debug:
     msg: "SSC pod status: {{ ssc_pod_info.resources[0].status.phase if ssc_pod_info.resources else 'No pods found' }}"
 
+- name: Deploy SSC LB
+  kubernetes.core.k8s:
+    state: present
+    template: templates/ssc-lb.yml.j2
+    kubeconfig: "{{ kubeconfig_file }}"
+  register: ssc_lb
+  when: expose_p2p
+
+- name: Get public IP from SSC LB
+  kubernetes.core.k8s_info:
+    kind: Service
+    name: ssc-public
+    namespace: sagasrv-ssc
+    wait: true
+    kubeconfig: "{{ kubeconfig_file }}"
+  until: (svc_info['resources'][0]['status']['loadBalancer']['ingress'][0]['hostname'] | default('')) != ''
```

Comment:
This only works on AWS. So, for the condition here you can remove `['hostname']`. Then below you need two set_fact. So, maybe just add
```
- name: Set SSC public address or localhost
  set_fact:
    ssc_external_address: "{{ svc_info['resources'][0]['status']['loadBalancer']['ingress'][0]['ip'] if expose_p2p else '127.0.0.1' }}"
    ```